server/src/permissions.py

```diff
@@ -0,0 +1,50 @@
+from aiohttp import web
+from routes import setup_routes
+import subprocess
+import config
+import signal
+import os
```

Comment:
This file seems like mostly a dupe of `src/main.py`. Can you apply the changes (line 6, line 33, line 42) you made here to `src/main.py` instead? And then rename the file `permissions`? That way we can preserve the history of that file.

Reply:
Ah, I see- `main.py` has the setup functions at the bottom. In that case, I think part of this PR can be removing all the duplicate stuff in that file and, instead, just importing `permissions.py` like you do in `src/gui.py`

Reply:
note to self: rename permission functions currently in permissions.py and gui.py to gui_*, add in functions from main.py to permissions.py, source permissions.py in main.py